### DISCUSSION TITLE
Lighthouse warning about preload with <Link> prefetch

### DISCUSSION THREAD

**Discussion by olaj** [vercel/next.js#18287] [Help]
I'm trying to improve the page speed score in my project and are getting pretty good numbers now. But this warning i get about preloaded links not being used has bugged me for a long time now.

The same or similar issues has been discussed before.

https://github.com/vercel/next.js/discussions/16961

I can get rid of this warning if i follow the tip in the above discussion about setting prefetch to false on <Links> but as i understand it this will not actually improve the performance but only "trick lighthouse" to give you a better score? 

https://github.com/vercel/next.js/issues/11215

In this thread its said that this was fixed in 9.5+ but i still get the issue with 9.5.

This problem is reproducible in for example this blog starter i found:

https://github.com/JoseRFelix/nextjs-starter-blog

Just run build / start then run lighthouse and you get the warning. If you set all <Link> to prefetch false and run build / start again then you do not get the warning in lighthouse.

But i have tried to reproduce it with for example

https://github.com/vercel/next.js/tree/canary/examples/blog-starter-typescript

but there it seems to work 😕 i can't figure out what the difference is with these projects? They have differences of course but i don't even know where to start looking :)

Thankful for any help / tips!

**Comment by olaj** [comment]
I'm still having this issue.

If anyone want to take a look you see the warning on this page (and well, all other pages as well).

https://alternativeto.net/software/stackoverflow/

I have no idea where to start looking to fix this issue but it would be really nice to get rid of it.

**Reply by RobSchilderr** [reply]
Hey I am having the same issue at next "10.0.5"! Following the thread. The issue was also posted here: https://github.com/vercel/next.js/issues/16932

**Reply by olaj** [reply]
https://github.com/GoogleChrome/lighthouse/discussions/12103#discussioncomment-381214
